myapp/views.py

```diff
@@ -61,3 +65,97 @@ def homework4(request):
     content['task6'] = Comment.objects.filter(content_type_id=article_model_id).order_by('date', '-article__author')[:2]
 
     return render(request, 'homework4.html', content)
+
+
+def check_requirements_form_view(request):
+    if request.method == 'POST':
+        form = CheckRequirementsForm(request.POST)
+        if form.is_valid():
+            name = request.POST['name']
+            sex = request.POST['sex']
+            age = int(request.POST['age'])
+            level = request.POST['english_level']
+
+            is_fit = (sex == 'm' and age >= 20 and level in ['C1', 'C2']) or \
```

Comment:
The correct way is to check values in form validation. It's SOLID, validation is a job of form class

Reply:
has moved